server/controllers/applicationsController.ts

```diff
@@ -174,10 +179,46 @@ const updateApplication = async (
   }
 };
 
+const getAggregatedUserStats = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const { userId } = req.params;
```

Comment:
Similar to comment on `applicationRoutes.ts` this works if we're letting people search other users stats, but is that the goal of this feature?

Reply:
Happy to discuss my current ideas for MVP features but one is that people can opt out of privacy and make their tracker available to everyone and not just in the aggregate data. As well - I want a toggle that allows users to see their aggregate data and the aggregate data for the entire user base and be able to make comparisons.